main.go

```diff
@@ -59,9 +62,41 @@ func validateRequirements(dataplaneBin, haproxyBin string) error {
 	return nil
 }
 
+func validateLogAddress(logAddress string) error {
+	fi, err := os.Stat(logAddress)
+	if err != nil {
+		// maybe it's a syslog host[:port]
+		if len(strings.Split(logAddress, ":")) == 1 {
+			// only host - default syslog port is 514 UDP
+			logAddress += ":514"
+		}
+		if govalidator.IsDialString(logAddress) {
+			conn, err := net.Dial("udp", logAddress)
```

Comment:
This is overkill and will be unexpected by users. It's the user's responsibility to provide a correct syslog address. We validate the format to ensure nothing is obviously wrong but a connectivity check here seems like too much.

Reply:
I had doubts about this too, so I am dropping this without regret. :)